src/services/blackMarketApi.ts

```diff
@@ -0,0 +1,20 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/dist/query/react';
+import { prepareHeaders } from 'utils/serviceUtils';
+
+export const blackMarketApi = createApi({
+  reducerPath: 'blackMarketAPI',
+  baseQuery: fetchBaseQuery({
+    baseUrl: process.env.REACT_APP_BASE_URL,
+    prepareHeaders,
+  }),
+  endpoints: (builder) => ({
+    postRegistration: builder.mutation({
```

Comment:
I think `createUser` is a more suitable name for this mutation, what do you think? Same for the one below, just `logOut` makes more sense to me